src/codegen/expression/null_translator.cpp

```diff
@@ -0,0 +1,45 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// null_translator.cpp
+//
+// Identification: src/codegen/expression/null_translator.cpp
+//
+// Copyright (c) 2015-2017, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "codegen/expression/null_translator.h"
+#include "codegen/type/boolean_type.h"
+#include "codegen/type/type_system.h"
+#include "expression/operator_expression.h"
+
+namespace peloton {
+namespace codegen {
+
+// Constructor
+NullTranslator::NullTranslator(const expression::OperatorExpression &null_check,
```

Comment:
Change the name of this class to something more descriptive like `NullCheckTranslator`